src/Symfony/Component/Yaml/CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 CHANGELOG
 =========
 
+3.2.5
```

Comment:
That's still not right. New features are only added in minor versions. So this needs to be `3.3`.